### ISSUE TITLE
Alias errors when used as `@rustbot author: xxx`

### ISSUE THREAD

**Issue by fee1-dead** [rust-lang/triagebot#1560]
Found in https://github.com/rust-lang/rust/pull/92892#issuecomment-1013335278, I wanted to write what exact parts are waiting on author so I wrote `@rustbot author: xxx` which caused an error.

**Comment by camelid** [comment]
FWIW, you can just write the explanation on a separate line, though I suspect you're opening this issue because you think the `:` style should be supported too :)

**Comment by ezekielathome** [comment]
This does not appear to be an issue anymore.
Input tested on `aa559cc` (commit near issue date):
```
Err(
    Error {
        input: "author: please add a ui test for the issue, in `src/test/ui/consts`, or in `src/test/ui/rfc-2632-const-trait-impl`, after that the changes LGTM.",
        position: 6,
        source: ExpectedEnd,
    },
)
```
Input tested on `9ea655c` (current master):
```
Ok(
    Some(
        Author,
    ),
)
```

**Comment by ehuss** [comment]
Indeed, I believe this is fixed by #1640, thanks!
